packages/multimedia/zvbi/package.mk

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2020-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="zvbi"
+PKG_VERSION="0.2.35"
+PKG_SHA256="fc883c34111a487c4a783f91b1b2bb5610d8d8e58dcba80c7ab31e67e4765318"
+PKG_LICENSE="GPL"
+PKG_SITE="http://zapping.sourceforge.net/ZVBI"
+PKG_URL="http://mirrors.kodi.tv/build-deps/sources/${PKG_NAME}-${PKG_VERSION}.tar.bz2"
+PKG_DEPENDS_TARGET="toolchain"
+PKG_LONGDESC="The Zapping VBI library, in short ZVBI, provides functions to capture and decode VBI data."
```

Comment:
```suggestion
PKG_LONGDESC="Library to provide functions to capture and decode VBI data."
```

Reply:
When possible I tend to use the same description used by the library itself, assuming it's adequate - ie. "in its own words". :smile:

Reply:
yea but in this case it is 50% pure spam ;)